pkg/utils/wildcard_test.go

```diff
@@ -0,0 +1,110 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestMatchWildcard(t *testing.T) {
```

Comment:
I'm slightly confused by the tests because the way _we_ are using `TestMatchWildcard` is against stack name patterns, no? While all these tests are against filename patterns. Technically, it's all the same; a string is a string. But I think we should add some tests for stack name patterns (e.g. `*-dev-*` or `*-{dev,staging}-*`

Reply:
Hrm... ok, so I see we're in `pkg/utils` so this is generic wildcard testing. Makes sense. But let's add still the patterns above, if we can.